pkg/gofr/datasource/redis/redis.go

```diff
@@ -63,6 +63,15 @@ func NewClient(c config.Config, logger datasource.Logger, metrics Metrics) *Redi
 	return &Redis{Client: rc, config: redisConfig, logger: logger}
 }
 
+// Close shuts down the Redis client, ensuring the current dataset is saved before exiting.
+func (r *Redis) Close(_ context.Context) error {
+	if r.Client != nil {
+		return r.Client.Close()
```

Comment:
Could you please explain why you changed from shutdown to close?

I'm OK with it,if it makes sense. I'm just asking.

Please rewrite the code to remove the context from the signature, and use the shutdowner method you already used for other closer without context then

Reply:
The Close method is to close the client, the shutdown was closing and shutting down the actual instance of redis.

Reply:
I have refactored the code and removed the context from signature